codeforamerica/vita-min
# re-encrypts staging credentials with new key

## Link to pivotal/JIRA issue
n/a

## Is PM acceptance required? (delete one)
No - merge after code review approval

## What was done?
- Re-encrypted the staging env creds w/ a new encryption key
- Added the updated key to lastpass
## How to test?
- Pull the branch
- Copy the new encryption key from lastpass into `config/credentials/staging.key`
- Ensure you can decrypt the file
